File: src/Suframa.Sciex.CrossCutting.DataTransferObject/ViewModel/QuadrosAdministradoresVM.cs
using System.Collections.Generic;

namespace Suframa.Sciex.CrossCutting.DataTransferObject.ViewModel
{
	public class QuadrosAdministradoresVM
	{
		public bool HasQuadroSocietario { get; set; }
		public int? IdNaturezaJuridica { get; set; }
		public int? IdPessoaJuridica { get; set; }
		public IEnumerable<NaturezaQualificacaoVM> NaturezaQualificacao { get; set; }
		public IEnumerable<QuadroAdministradoresVM> QuadrosAdministradores { get; set; }
	}
}

File: src/Suframa.Sciex.UserInterface/www/app/view-model/QuadrosAdministradoresVM.ts
import { quadroAdministradoresVM } from './QuadroAdministradoresVM';
import { naturezaQualificacaoVM } from './NaturezaQualificacaoVM';

export class quadrosAdministradoresVM {
	constructor() {
		this.naturezaQualificacao = Array<naturezaQualificacaoVM>();
		this.quadrosAdministradores = Array<quadroAdministradoresVM>();
	}

	idPessoaJuridica?: number;
	idNaturezaJuridica?: number;
	hasQuadroSocietario?: boolean;
	naturezaQualificacao: naturezaQualificacaoVM[];
	quadrosAdministradores: quadroAdministradoresVM[];
}
